bin/cws-shelf-upload.js

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env node
+"use strict";
+
+var container;
+
+container = require("../lib/container");
+container.callAsync((config, helpMessage, neodoc) => {
+    var args;
+
+    args = neodoc.run(helpMessage, config.neodocOptions);
+
+    // Set debug in the config before we load more modules
+    config.debug = !!args["--debug"];
```

Comment:
Hmm I am a little hesitant. That is fine for tools we use since the only people using them are the people who wrote them (or at least they are part of that team). This might cause more confusion than it helps. For example if they already had DEBUG set for something else.

@fidian Do you have an opinion on this?